_data/hosting.yml

```diff
@@ -95,6 +95,14 @@ websites:
       tfa: No
       twitter: ovh_support_en
 
+    - name: PlanetHoster
+      url: https://planethoster.net
+      twitter: PlanetHoster
```

Comment:
One last thing, could you remove the twitter handle?
Twitter isn't needed for sites already supporting TFA.